components/bl-alert-component/README.md

```diff
@@ -0,0 +1,46 @@
+# Alert
```

Comment:
add screenshots please, and don't forget to check all points in checklist https://github.com/Backendless/ui-builder-library/wiki/PR-Checklist